plugins/aws/aws.plugin.zsh

```diff
@@ -202,8 +202,17 @@ function aws_change_access_key() {
 }
 
 function aws_regions() {
+  local -a region
```

Comment:
Why is this an array? Shouldn't be a regular variable?

Reply:
You're right, sorry.
It's fixed.